src/app/OperationalDeviceProxy.cpp

```diff
@@ -169,7 +169,8 @@ void OperationalDeviceProxy::UpdateDeviceData(const Transport::PeerAddress & add
     char peerAddrBuff[Transport::PeerAddress::kMaxToStringSize];
     addr.ToString(peerAddrBuff);
 
-    ChipLogDetail(Controller, "Updating device address to %s while in state %d", peerAddrBuff, static_cast<int>(mState));
+    ChipLogDetail(Controller, "Updating device address to %s while in state %d for node 0x" ChipLogFormatX64, peerAddrBuff,
```

Comment:
NodeId alone is insufficient here - please use the format mentioned much earlier in this file:

`OperationalDeviceProxy[" ChipLogFormatX64 ":" ChipLogFormatX64 "]:`, which logs the PeerId + NodeId.